data-loader/cli/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM eclipse-temurin:8-jre-jammy
+
+RUN apt-get update && apt-get upgrade -y \
```

It has become a Docker-recommended practice to add this. 
cf. https://github.com/scalar-labs/scalardb-cluster/pull/1061
```suggestion
RUN apt-get update && apt-get upgrade -y --no-install-recommends \
```